### ISSUE TITLE
Confusing "Checkboxes" and "Multiple choices" naming

### ISSUE THREAD

**Issue by 42l-neil-tmp** [nextcloud/forms#597]
**Nextcloud**:
- Nextcloud-Version: 19.0.2
- Forms-Version: 2.0.4

Related to: Frontend, UX, i18n

Hi,

During UX tests, some people are confused with the name of the "Multiple choice" and "checkboxes" question types:
![image](https://user-images.githubusercontent.com/70517536/92585960-ec7ec000-f295-11ea-9959-019663ef0a5c.png)

- "Multiple choice" implies that the user can select multiple answers, while it isn't case – radio buttons do not allow that.
- "Checkboxes" is actually the real "multiple choices" question type, but its technical term is used, which might not be desirable.

Here's a proposition:
- Rename "Checkboxes" to "Multiple choices"
- Rename "Multiple choices" (radio buttons) to "single choice"

I'm not native English so if you have better ideas, please suggest. How are Google Forms / Typeform folks doing this (and is it intuitive) ?

Thanks for reading!

**Comment by skjnldsv** [comment]
cc @jancborchardt

**Comment by jotoeri** [comment]
Hey @42l-neil-tmp,

as it seems you are mainly working in french on forms, no?
The term 'multiple choice' is the right term for choosing only one option in english. ("Ils sont fous, ces Anglais!"?) So it might be interesting for you to change the french translation on transifex (https://www.transifex.com/nextcloud/nextcloud/forms/) to a better version. The German translation already translates as 'single choice', even that this is not the translation by word, but it is by sense.

Greets,
Jonas

Still, @jancborchardt Might it be useful to search for another term on source-language, to avoid confusion on (quite all?) other languages?

**Comment by 42l-neil-tmp** [comment]
That's right, I'm working in French :stuck_out_tongue: 

Actually the person who reported me this is French but was using Nextcloud Forms in English. It makes sense though, she was probably confused by the English version (I was too!), since that's not how we would say it in our language.
The French translation has transcribed the same ambiguity, though.

I submitted a suggestion to edit the French translation in Transifex (and spent my day translating all the untranslated strings in the Nextcloud Transifex board to French!), waiting for approbation from the coordinators/reviewers.

I saw the comments in Transifex on those entries weren't unanimous on this matter as well, but feel free to close if there is no other appropriate term native speakers can think of.
Thanks for the clarification!
